chef_master/source/cookbook_repo.rst

```diff
@@ -116,7 +116,7 @@ This configuration file has the following settings:
         :default => 'kitty kitty'
 
 ``chef_version``
-   A range of chef-client versions that are supported by this cookbook.
+   A range of chef-client versions that are supported by this cookbook. All :ref:`version constraint operators <cookbook_version_constraints>` are applicable to this field.
```

Comment:
And actually we don't really doc the multi constraint version on this page, so that's fine.